.shellcheckrc

```diff
@@ -0,0 +1,5 @@
+# NOTE: these disables are only necessary while the core24 spread tests are
```

Comment:
do we need to remove this?

Reply:
probably, yes. I added this because of all the early `exit` we put in while the features were being ported over